src/app/results/results.component.ts

```diff
@@ -43,8 +47,22 @@ export class ResultsComponent implements OnInit {
     return result;
   };
   advancedsearch() {
-
   }
+  // display content on hover
+  // --------------------------------
+  overTitle(){
+    if(this.hoverBox == true){
+      this.hoverBox = false;
+    }
+    else {
```

Comment:
try something like this:
if(condition){
    //code
}else{
   //code
}
maybe this will fix the lint error